Pokepedia.Api/Controllers/Pokemons/PokemonGet/LocationByPokemonNameController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Pokepedia.Api.Controllers.Pokemons.PokemonGet.Requests;
+using Pokepedia.Api.Controllers.Pokemons.PokemonGet.Responses;
+using Pokepedia.Domain.Entities.Pokemons;
+using Pokepedia.Domain.Services.Crud.Read;
+using Pokepedia.Domain.Validation;
+
+namespace Pokepedia.Api.Controllers.Pokemons.PokemonGet
+{
+    [ApiController]
+    [Route("v1")]
+    public class LocationByPokemonNameController : ControllerBase
```

Comment:
if this is location by pokemon name. This shouldnt be in the PokemonGet folder but the LocationGet folder instead